### PR TITLE
feat(clerk-js): Adjust captcha parameter handling for sign ups with Google

### PR DESCRIPTION
**Pull request by BRKalow**
## Description

Adjust how we pass captcha parameters when signing up via Google.

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

**Comment by BRKalow** [comment]
!snapshot

File: .changeset/rude-wasps-wonder.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@clerk/clerk-js": patch
+---
+
+Adjust how we pass captcha tokens to the Clerk API when signing in with Google.
```
Comment (thiskevinwang) [inline .changeset/rude-wasps-wonder.md:5]:
```suggestion
Adjust how we pass captcha tokens to the Clerk API when signing in with Google, Microsoft, and Apple
```

re: a81f7ce1240afe5b64a25058456564b888692a01

**Comment by BRKalow** [comment]
!snapshot

**Comment by BRKalow** [comment]
!snapshot

**Comment by BRKalow** [comment]
!snapshot

**Comment by clerk-cookie** [comment]
Hey @BRKalow - the snapshot version command generated the following package versions:
| Package | Version |
| --- | --- |
| @clerk/astro | 1.0.4-snapshot.vcfba6d4 |
| @clerk/chrome-extension | 1.1.7-snapshot.vcfba6d4 |
| @clerk/clerk-js | 5.10.2-snapshot.vcfba6d4 |
| @clerk/dev-cli | 0.0.3-snapshot.vcfba6d4 |
| @clerk/clerk-expo | 1.2.10-snapshot.vcfba6d4 |
| gatsby-plugin-clerk | 5.0.0-beta.45 |

Tip: Use the snippet copy button below to quickly install the required packages.
`@clerk/astro`
```sh
npm i @clerk/astro@1.0.4-snapshot.vcfba6d4 --save-exact
```
`@clerk/chrome-extension`
```sh
npm i @clerk/chrome-extension@1.1.7-snapshot.vcfba6d4 --save-exact
```
`@clerk/clerk-js`
```sh
npm i @clerk/clerk-js@5.10.2-snapshot.vcfba6d4 --save-exact
```
`@clerk/dev-cli`
```sh
npm i @clerk/dev-cli@0.0.3-snapshot.vcfba6d4 --save-exact
```
`@clerk/clerk-expo`
```sh
npm i @clerk/clerk-expo@1.2.10-snapshot.vcfba6d4 --save-exact
```
`gatsby-plugin-clerk`
```sh
npm i gatsby-plugin-clerk@5.0.0-beta.45 --save-exact
```

**Review by thiskevinwang — APPROVED** [review]

**Review by BrandonRomano — APPROVED** [review]

**Pull request merged**